mdpow/forcefields.py

```diff
@@ -141,13 +141,25 @@ def get_water_model(watermodel=DEFAULT_WATER_MODEL):
         raise ValueError(msg)
 
 #: Other solvents (not water, see :data:`GROMACS_WATER_MODELS` for those).
-OPLS_SOLVENT_MODELS = {
+OPLS_OCTANOL_MODELS = {
     'octanol': GromacsSolventModel(
         identifier="octanol", itp="1oct.itp", coordinates="1oct.gro"),
-    'cyclohexane': GromacsSolventModel(
-        identifier="cyclohexane", itp="1cyclo.itp", coordinates="1cyclo.gro"),
+    'octanolnew': GromacsSolventModel(
+        identifier="octanol", itp="1octnew.itp", coordinates="1oct.gro")
+    }
+
+OPLS_WETOCTANOL_MODELS = {
     'wetoctanol': GromacsSolventModel(
         identifier="wetoctanol", itp="1octwet.itp", coordinates="1octwet.gro"),
+    'wetoctanolnew': GromacsSolventModel(
```

Comment:
add a comment what "new" means, i.e., the citation for the parameters – "new" doesn't mean anything to anyone looking at this

Reply:
Zangi R (2018) Refinement of the OPLSAA force-field for liquid alcohols. 
; ACS Omega 3(12):18089–18099. doi: 10.1021/acsomega.8b03132

Reply:
I put the citation into ``description`` attribute of the class.